docs-2.0/3.ngql-guide/16.subgraph-and-path/2.find-path.md

```diff
@@ -0,0 +1,53 @@
+# FIND PATH
+
+```ngql
+FIND SHORTEST | ALL | NOLOOP PATH FROM <vertex_id_list> TO <vertex_id_list>
+OVER <edge_type_list> [REVERSELY] [BIDIRECT] [UPTO <N> STEPS]
+
+<vertex_id_list> ::=
+    [vertex_id [, vertex_id ...]
+```
+
+The `FIND PATH` statement finds the paths between the selected source vertex and destination vertex.
+
+- `SHORTEST` finds the shortest path.
+- `ALL` finds all the paths.
+- `<vertex_id_list>` is a list of vertex IDs separated with commas (,). It supports `$-` and `$var`.
+- `<edge_type_list>`  is a list of edge types separated with commas (,). `*` is all edge types.
+- `<N>` is the hop number. The default value is 5.
+
+## Limitations
+
+- When a list of source and/or destination vertex IDs are specified, the shortest path between any source vertices and the destination vertices is returned.
+- There can be cycles when searching all paths.
+- `FIND PATH` does not support property filtering.
+- `FIND PATH` does not support specifying direction.
+- `FIND PATH` is a single-process procedure, so it uses much memory.
+- When finding path with the `NOLOOP` keyword, the returned paths do not include cycles.
+
+## Examples
+
+In Nebula Console, path is shown as `vertex_id <edge_name, rank> vertex_id`.
+
+```ngql
+nebula> FIND SHORTEST PATH FROM "player102" TO "team201" OVER *;
++------------------------------------------------------------------+
+| path                                                             |
++------------------------------------------------------------------+
+| ("player102")-[:follow@0]->("player101")-[:serve@0]->("team201") |
++------------------------------------------------------------------+
+```
+
+```ngql
+nebula> FIND SHORTEST PATH FROM "player100" TO "team200" OVER * REVERSELY;
+Empty set (time spent 17442/18406 us)
+```
```

Comment:
should we give some explanation about the "Empty set" result?

Reply:
I’m not sure. But if it's necessary I don’t think here is a good idea

Reply:
> I’m not sure. But if it's necessary I don’t think here is a good idea

Why do you give such an example. When I read an example that gives an empty result, I suppose you have some special purpose. SO I have the preceding question.